cmd/loadTests.go

```diff
@@ -200,17 +201,24 @@ func setup(cmd *cobra.Command, args []string) {
 			userIndex := AppStudioUsersBar.Current()
 			startTime := time.Now()
 			username := fmt.Sprintf("%s-%04d", usernamePrefix, userIndex)
-			if err := users.Create(framework.AsKubeAdmin.CommonController.KubeRest(), username, constants.HostOperatorNamespace, constants.MemberOperatorNamespace); err != nil {
+			framework, err := framework.NewFramework(username)
+			if err != nil {
+				fmt.Println(err.Error())
+				os.Exit(1)
+			}
```

Comment:
Please,do not use os.Exit(1) use the klog.Fatalf instead to keep it consistent with the rest of the code
```suggestion
			if err != nil {
				klog.Fatalf("Unable to create new framework for user %s: %v", username, err)
			}
```